addon/initializers/m3-store.js

```diff
@@ -56,6 +59,18 @@ export function extendStore(Store) {
       return this._super(modelName, id, clientId, storeWrapper);
     },
 
+    /**
+     * A thin wrapper around the API response that knows how to look up relationships.
```

Comment:
I'd be tempted to change `relationships` to `references`.  In practice they serve similar functionality but they have different semantics so I'd rather use a different term in an attempt to avoid confusing people.